src/components/AcceptOrRejectAll.tsx

```diff
@@ -0,0 +1,62 @@
+// external
+import { h, JSX } from 'preact';
+import { useIntl } from 'react-intl';
+
+// global
+import { useAirgap } from '../hooks';
+import { messages } from '../messages';
+import type { HandleSetViewState } from '../types';
+
+// local
+import Button from './Button';
+import ColumnContent from './ColumnContent';
+import Title from './Title';
+
+/**
+ * Component showing "accept all" or "reject all"
+ */
+export default function AcceptOrRejectAll({
+  handleSetViewState,
+}: {
+  /** Function to change viewState */
+  handleSetViewState: HandleSetViewState;
+}): JSX.Element {
+  const { airgap } = useAirgap();
+  const { formatMessage } = useIntl();
+
+  // Opt in to all purposes
+  const handleAcceptAll:
+    | JSX.MouseEventHandler<HTMLButtonElement>
+    | undefined = (
+    event: JSX.TargetedEvent<HTMLButtonElement, MouseEvent>,
+  ): void => {
+    event.preventDefault();
+    airgap.optIn(event);
+    handleSetViewState('close');
+  };
+
+  // Opt out of all non-essential purposes
+  const handleRejectAll:
+    | JSX.MouseEventHandler<HTMLButtonElement>
+    | undefined = (
+    event: JSX.TargetedEvent<HTMLButtonElement, MouseEvent>,
+  ): void => {
+    event.preventDefault();
+    airgap.optOut(event);
+    handleSetViewState('close');
+  };
+
+  return (
+    <ColumnContent>
+      <Title align="left">{formatMessage(messages.consentTitle)}</Title>
```

Comment:
OK to merge but this message isn't ready for prime time.

Reply:
do you have a preferred message? 

we are also adding message overrides for customers to change it as they desire

Reply:
(routing to legal)